docs/sql-guide/functions/function-parsetimestamp.md

```diff
@@ -0,0 +1,138 @@
+---
+title: PARSETIMESTAMP()
+layout: default
+parent: SQL functions
+grand_parent: SQL guide
+---
+## Before you begin
+* [Learn about unix epoch/unix time](https://en.wikipedia.org/wiki/Unix_time){:target="_blank"}
+
+# PARSETIMESTAMP() function
+
+The `PARSETIMESTAMP()` function converts the input time value in string format to timestamp based on the format provided.
+
+## Syntax
+
+```
+PARSETIMESTAMP(expr, format)
+```
+
+## Arguments
+
+| Argument | Data type | Description | Required | Further information |
+|---|---|---|---|---|
+| expr   | string | Input string values that needs to be converted to a timestamp | Yes |
```

Comment:
You need a last pipe character on the end of the `expr` row otherwise the table will fall apart.